src/basic/cgroup-util.c

```diff
@@ -2126,6 +2126,7 @@ static const char *const cgroup_controller_table[_CGROUP_CONTROLLER_MAX] = {
         [CGROUP_CONTROLLER_PIDS] = "pids",
         [CGROUP_CONTROLLER_BPF_FIREWALL] = "bpf-firewall",
         [CGROUP_CONTROLLER_BPF_DEVICES] = "bpf-devices",
+        [CGROUP_CONTROLLER_BPF_PINNED_PROG] = "bpf-pinned-prog",
```

Comment:
i am not sure i like the word "pinned"... Maybe call this "bpf-foreign" or so?

(also not a fan of needless abbreviation, i.e. prog vs. program)

Reply:
@poettering
 
I think `foreign` is not descriptive. Wdyt about `bpf-fs-prog`: short and easy?

Reply:
but they all might end up in bpffs sooner or later, even if we compile them ourselves, no? key really is that we don't generate these programs ourselves, but someone else did, no? hence my proposal "foreign".

i mean, whether these progs show up in bpffs or not defining, it is that we just use what somebody else already put there.

other ideas for naming. "import", "preloaded", "external" or so...

Reply:
@poettering

`Foreign` it is! :)